processor/src/chiplets/tests.rs

```diff
@@ -3,56 +3,56 @@ use vm_core::{
     bitwise::{BITWISE_OR, OP_CYCLE_LEN},
     hasher::{HASH_CYCLE_LEN, LINEAR_HASH, RETURN_STATE},
     program::blocks::CodeBlock,
-    Felt, FieldElement, ProgramInputs, AUX_TABLE_RANGE, AUX_TABLE_WIDTH,
+    Felt, FieldElement, ProgramInputs, CHIPLETS_RANGE, CHIPLETS_WIDTH,
 };
 
-type AuxTableTrace = [Vec<Felt>; AUX_TABLE_WIDTH];
+type ChipletsTrace = [Vec<Felt>; CHIPLETS_WIDTH];
 
 #[test]
 fn hasher_aux_trace() {
     // --- single hasher permutation with no stack manipulation -----------------------------------
     let stack = [2, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0];
     let operations = vec![Operation::RpPerm];
-    let (aux_table_trace, trace_len) = build_trace(&stack, operations);
+    let (chiplets_trace, trace_len) = build_trace(&stack, operations);
```

Comment:
I agree it doesn't sound great. I debated it a bit, but leaned toward what I thought was clearer. (This is also why I ended up using the phrase "chiplets module" in documentation references to specify the set of all the chiplets.) I was really on the fence though, so I'm fine with changing this.

Reply:
I take it back - I'm going to leave it as it is. If we change this, it would also make sense to change to chiplet_bus, etc, and I think it's clearer to use the plural in cases where we're referring to the whole module and all of the chiplets. We'll just have to imagine in an apostrophe to satisfy our internal grammarians :)